packages/ui/components/filter-select/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { classNames } from "@calcom/lib";
+import {
+  Button,
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  Command,
+  CommandInput,
+  CommandList,
+  CommandEmpty,
+  CommandGroup,
+  CommandItem,
+  CommandSeparator,
+  Badge,
+  Icon,
+} from "@calcom/ui";
+
+export interface FilterOption {
+  value: string | number;
+  label: string;
+  icon?: React.ReactNode;
+}
+
+interface FilterSelectProps {
+  title: string;
+  options: FilterOption[];
+  selectedValue?: string | number | null;
+  onChange: (value: string | number | null) => void;
+  buttonIcon?: React.ReactNode;
+  placeholder?: string;
+  emptyText?: string;
+}
+
+export function FilterSelect({
```

Comment:
Reusable style for filters based on what we used in data-table but not being tanstack table dependant